src/components/uploads/dialog/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @author johnworth
```

Comment:
Picking a random spot to respond to Sriram's question - I'm not personally used to seeing index files, so I would prefer file names that indicate the purpose.  That's just my opinion though.